traffic_ops/traffic_ops_golang/riaksvc/riak_services.go

```diff
@@ -219,15 +227,16 @@ WHERE t.name = 'RIAK' AND st.name = 'ONLINE'
 	return servers, nil
 }
 
-func RiakServersToCluster(servers []ServerAddr, authOptions *riak.AuthOptions) (StorageCluster, error) {
+func riakServersToClusterPriv(servers []ServerAddr, authOptions *riak.AuthOptions) (*riak.Cluster, error) {
 	if authOptions == nil {
 		return nil, errors.New("ERROR: no riak auth information from riak.conf, cannot authenticate to any riak servers")
 	}
 	nodes := []*riak.Node{}
 	for _, srv := range servers {
 		nodeOpts := &riak.NodeOptions{
-			RemoteAddress: srv.FQDN + ":" + srv.Port,
-			AuthOptions:   authOptions,
+			RemoteAddress:       srv.FQDN + ":" + srv.Port,
+			AuthOptions:         authOptions,
+			HealthCheckInterval: HealthCheckInterval,
```

Comment:
This field doesn't appear to actually be used by the client: https://github.com/basho/riak-go-client/search?q=HealthCheckInterval&unscoped_q=HealthCheckInterval 

??

Reply:
@rob05c The default node manager, see node.go, kicks off health checks on each node in the cluster and will removed one from use until it's healthy again.  I didn't think it was necessary to have a go routine that queries the database and updates cluster membership as I don't think this happens enough to warrant that.  as long as the default node manager picks a healthy node to execute the commands on I think we're okay.  We'll probably restart more often than we add new riak nodes.